Library/Homebrew/formula_installer.rb

```diff
@@ -777,13 +777,18 @@ def pour
     end
 
     keg = Keg.new(formula.prefix)
+    tab_file = formula.prefix.join(Tab::FILENAME)
+    # Skip the cache since the receipt will be rewritten
```

Comment:
Sure. `Tab.from_file` and `Tab.for_keg` both cache the resulting `Tab` instance so that when either is called with the same path, the same instance is returned. `Tab.from_file_content` bypasses the cache. This is desirable here because `keg.relocate_text_files` will rewrite the tab file, and we want to load in the rewritten file before modifying it and writing it out again a few lines down.